plugins/ci/pkg/ci/ci.go

```diff
@@ -535,17 +537,100 @@ func ProcessHelmTemplates(helmConfigs []models.HelmConfig, tempFolder string, co
 			if err != nil {
 				return err
 			}
+		} else if helm.IsFluxFile() {
```

Comment:
there's something off about the indentation here 🤔 maybe it's just GitHub

Reply:
Looks like GitHub is to blame on this one

<img width="1038" alt="Screen Shot 2021-10-29 at 17 24 48" src="https://user-images.githubusercontent.com/2960590/139497553-49d576ff-4648-46be-a69f-c2847c11bfdc.png">